DigitalOffice/Tests/UserServiceTests/ValidatorsTests/UserCredentialsValidatorTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿using FluentValidation;
+using NUnit.Framework;
+using UserService.Database.Entities;
+using UserService.Models;
+using UserService.Validators;
+
+namespace UserServiceUnitTests.ValidatorsTests
+{
+    class UserCredentialsValidatorTests
+    {
+        public class Tests
+        {
+            private IValidator<UserCredentials> validator = new UserLoginValidator();
+
+            [Test]
+            public void EmptyLoginEmptyPasswordTest()
+            {
+                var creds = new UserCredentials
+                {
+                    Email = "",
+                    Password = ""
+                };
+
+                Assert.Throws<ValidationException>(() => validator.ValidateAndThrow(creds), "Email is empty.");
```

Comment:
Those construction can be simplified to
validator.ShouldHaveValidationErrorFor()

Reply:
fixed